src/Core/Components/Avatar/FluentAvatar.razor

```diff
@@ -0,0 +1,34 @@
+﻿@namespace Microsoft.FluentUI.AspNetCore.Components
+@using Microsoft.AspNetCore.Components.Rendering;
+@using Microsoft.FluentUI.AspNetCore.Components.Extensions
+@inherits FluentComponentBase
+
+<fluent-avatar id="@Id"
+               role="button"
```

Comment:
Only for `OnClick.HasDelegate`. By default, it is not a button. Same for `tabindex´